include/VotingArbiter.h

```diff
@@ -363,6 +366,15 @@ inline TimeKeeper VotingArbiter::getStartTime(void) const
   return _startTime;
 }
 
+struct PollOption {
+  std::string pollParameters;
+  bz_CustomPollOptionHandler* pollHandler;
+};
+extern std::map<std::string, PollOption> customPollOptions;
```

Comment:
Without making this extern, I wouldn't be able to access the map in other files as is. Do you have a better approach in mind for this?